pyproject.toml

```diff
@@ -0,0 +1,53 @@
+[build-system]
+requires      = ["setuptools>=61.0.0", "wheel"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "Autotype"
+version = "1.0.0"
+description = "A quick and small python script that helps you autotype on websites."
+readme = "README.md"
+authors = [{ name = "Tushar Gupta", email = "codingid6@gmail.com" }]
+license = { file = "LICENSE" }
+classifiers = [
+    "License :: OSI Approved :: MIT License",
+    "Programming Language :: Python",
+    "Programming Language :: Python :: 3",
+]
+keywords = ["Autotype", "simulator", "commandline"]
+dependencies = [
+    "pynput",
+    "typer",
+    "progressbar2",
+    "colorama",
+]
+requires-python = ">=3.9"
```

Comment:
Ubuntu 20.04 comes with python 3.8 as default. Are there any major reason for deprecating support for python 3.8 ?